filters/mermaidDiagram.js

```diff
@@ -55,8 +54,7 @@ function generateMermaidDiagram(asyncapi) {
     }
     seenSchema.push(schemaId);
   };
-
-  traversePayloadSchemas(asyncapi, generateDiagram);
+  asyncapi.traverseSchemas(generateDiagram);
```

Comment:
Is it supposed to not only run the code of the callback when it iterates a new schema but also when it is done iterating through a schema? i.e. dont you want to check for the callback type -> https://github.com/asyncapi/parser-js/blob/82832fbea46d6aae803fae54f889b6259f87140f/lib/models/asyncapi.js#L264

Reply:
why would I? I mean why would I differentiate?

Reply:
Because the callback is called twice per schema, but I guess you discard them it they already been generated, so all good in the end. Just wanted to hear 😄

Reply:
yeah, the output looks the same so all good 😄